geowebcache/core/pom.xml

```diff
@@ -205,6 +205,13 @@
       <artifactId>awaitility</artifactId>
       <scope>test</scope>
     </dependency>
+    <dependency>
+      <!-- used for tests that require environment variables -->
+      <groupId>com.github.stefanbirkner</groupId>
```

Comment:
interesting library, please move version number move up to dependency management section.

Reply:
[done](https://github.com/GeoWebCache/geowebcache/compare/e3c9c90f322d52293ad1fa383c6723bf3e3a9959..6634b038ff54ed9f89b1c8a144cbf227929843c0). Thanks